lib/mp-readline/readline.c

```diff
@@ -98,6 +99,38 @@ typedef struct _readline_t {
 
 STATIC readline_t rl;
 
+size_t get_previous_word_cursor_pos() {
```

Comment:
C requires `(void)` for functions without args.

Reply:
I haven't written C in a very long time and would like to believe that if it compiles it's correct :laughing:

Reply:
In C, `void foo()` says that the function foo can take any number of arguments, where `void foo(void)` says that the function foo takes no arguments. Both are syntactically correct but have  different semantic interpretations.